mandatory/2-function-creation.js

```diff
@@ -16,7 +23,13 @@ Write a function that:
 - numbers greater 100 must be replaced with 100
 */
 
-function formatPercentage(num) {}
+function formatPercentage(num) {
+  if(num >= 100){
+    return '100%';
+  } else {
+  return Number(num.toFixed(2)) + '%';
```

Comment:
Minor point, but I would be careful with the indentation here - just to improve the readability of the code.